modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -54,16 +58,6 @@ def __init__(self, data, length=None, width=None, ip=None, call_queue=None):
         self._ip_cache = ip
         self._identity = uuid.uuid4().hex
 
-        logger = get_logger()
```

Comment:
Is there a reason to drop debug logging? `ClassLogger` only logs calls/exits at `.info` and it doesn't log particular details which might be interesting. I would just change the logging from eager formatting to lazy one (specifying the message template and format args as separate things, like `logger.debug("My message: {height}", height=self._length_cache)`, so `logger` would compose the message only when debug is enabled.

Here and below.